requests/adapters.py

```diff
@@ -493,13 +493,12 @@ def send(self, request, stream=False, timeout=None, verify=True, cert=None, prox
                 # TODO: Remove this in 3.0.0: see #2811
                 if not isinstance(e.reason, NewConnectionError):
                     raise ConnectTimeout(e, request=request)
-
-            if isinstance(e.reason, ResponseError):
+            elif isinstance(e.reason, ResponseError):
```

Comment:
Why are these being changed to `elif` s? In the event that `e.reason` is a `ConnectTimeoutError` but not a `NewConnectionError` we'll now be unable to re-raise in the event that it's one of these other exeption types.